<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-8QPJLAI" name="Com.Testcases.NewBusiness.NewBusiness_01" tests="5" failures="1" timestamp="2 Mar 2020 14:08:40 GMT" time="18.880" errors="0">
  <testcase name="Test_NewBusiness_01" time="10.741" classname="Com.Testcases.NewBusiness.NewBusiness_01"/>
  <testcase name="Test_NewBusiness_01" time="0.957" classname="Com.Testcases.NewBusiness.NewBusiness_01"/>
  <testcase name="Test_NewBusiness_01" time="7.157" classname="Com.Testcases.NewBusiness.NewBusiness_01">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Com.Testcases.NewBusiness.NewBusiness_01.Test_NewBusiness_01(NewBusiness_01.java:566)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- Test_NewBusiness_01 -->
  <testcase name="Test_NewBusiness_01" time="0.014" classname="Com.Testcases.NewBusiness.NewBusiness_01"/>
  <testcase name="Test_NewBusiness_01" time="0.011" classname="Com.Testcases.NewBusiness.NewBusiness_01"/>
</testsuite> <!-- Com.Testcases.NewBusiness.NewBusiness_01 -->
